Bugsnag.NET.Tests/Request/NoticeTests.cs

```diff
@@ -24,6 +25,17 @@ public void HasCorrectSchema()
             Assert.IsTrue(jObject.IsValid(schema));
         }
 
+        [Test]
+        public void StackIsAbsent()
+        {
+            const string InternalErrorMessage = "An internal error has occurred!";
+            Exception ex = new Exception(InternalErrorMessage, new Exception("Debug Info: " + Environment.NewLine + "Some debug information goes here..."));
+            var ev = new Event(ex);
+            var notice = new Notice("apikey", new Notifier(), ev);
+            var json = JsonConvert.SerializeObject(notice);
+            // should reach the end of function without any exception thrown
```

Comment:
Would you mind wrapping the appropriate lines here in an `Assert.DoesNotThrow` call?